tests/integration/podman/scripts/setup_ssh_env.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+# This script creates an SSH server inside a container to test the kw ssh
+# feature.
+
+set -e
+
+function detect_distro()
+{
+  if [[ -f '/etc/os-release' ]]; then
+    . /etc/os-release
+    printf "%s\n" "$ID"
+  else
+    printf 'Distribution not detected\n'
+  fi
+}
+
+# This function configures the container storage environment. The primary
+# objective is to resolve issues related to the overlay storage driver on Arch
+# Linux systems. For Arch Linux, it sets the storage configuration to use an
+# alternative setup defined in archlinux_storage.conf and removes the
+# existing container storage directory
+function configure_container_environment()
+{
+  local distro
+  local podman_tmp_utils='/root/kw-ssh-tests'
+  local containers_config_dir='/etc/containers'
+  local arch_storage_conf="${podman_tmp_utils}/archlinux_storage.conf"
+  local containers_config_file="${containers_config_dir}/containers.conf"
+  local storage_config_file="${containers_config_dir}/storage.conf"
+
+  distro=$(detect_distro)
+
+  if [[ "$distro" =~ 'arch' ]]; then
+    printf "%s" "$(< ${arch_storage_conf})" > "$storage_config_file"
+    rm -rf '/var/lib/containers/storage'
+  fi
+}
+
+# Ensure /usr/sbin is in the PATH
+function ensure_path()
+{
+  printf 'Ensuring /usr/sbin is in the PATH...\n'
```

Comment:
Following my comments about "OK output", this also applies her (unless these are stored in a log file of sorts).